projects/dev/src/app/group/group-manage/pending-request/pending-request.component.ts

```diff
@@ -22,87 +37,149 @@ export class PendingRequestComponent implements OnInit, OnChanges {
     { field: "at", order: -1 },
     { field: "member_id", order: 1 },
   ];
-  prevSortMeta = "-at member_id";
-
-  @Output() onAccept = new EventEmitter<any>();
-  @Output() onReject = new EventEmitter<any>();
-
-  groupSwitch = [
-    {
-      label: "This group only",
-    },
-    {
-      label: "All subgroups",
-    },
-  ];
+  prevSortMeta: string = "-at member_id";
+
+  requestLoading: boolean = false;
+  requestAction: string = "";
```

Comment:
You still have several non independent variables! :-/

Reply:
fixed.